app/views/shared/_navbar.html.erb

```diff
@@ -1,8 +1,17 @@
 <nav class="navbar navbar-light bg-light">
   <div class="container-fluid d-flex justify-content-between">
-    <a class="navbar-brand" href="#">
-      <img src="/docs/5.0/assets/brand/bootstrap-logo.svg" alt="" width="30" height="24">
-    </a>
-    <img src="/docs/5.0/assets/brand/bootstrap-logo.svg" alt="" width="30" height="24">
+    <%= link_to root_path, class: "navbar-brand" do %>
+      <%= image_tag 'logo-blanco', class: "navbar-logo" %>
+    <% end %>
+    <% if current_user && current_user.admin = true %>
+      <%= link_to "Crear nuevo proveedor", new_provider_path %>
+    <% elsif current_user %>
+      <div class="d-flex">
+        <%= image_tag 'pedidos.png', input_html: {id: "pedidos-img"} %>
```

Comment:
why do we need this 
`input_html: {id: "pedidos-img"}`?

Reply:
Because we need to give that image some particular styles, so I added that ID, is it wrong?